### ISSUE TITLE
Add transaction confirmation view to replace modal

### ISSUE THREAD

**Issue by willbach** [urbit/star.market#32]

**Comment by g-a-v-i-n** [comment]
Here are a few mocks - the idea is to replace the yellow swap 'preview' with this  green confirmation version once the transaction succeed.

<img width="531" alt="Screen Shot 2021-11-03 at 12 38 32 PM" src="https://user-images.githubusercontent.com/5265021/140103789-93faf5b6-eb10-4904-a276-42b2c6580ee7.png">

<img width="527" alt="Screen Shot 2021-11-03 at 12 38 27 PM" src="https://user-images.githubusercontent.com/5265021/140103791-2f025f7b-620a-4291-8b70-5f317fc6e0af.png">

@urcades @marisarow Any thoughts or feedback?

**Comment by urcades** [comment]
@g-a-v-i-n yes, this is a huge improvement on the mocks originally posted, I support its structure
